src/main/java/com/wynntils/modules/utilities/configs/UtilitiesConfig.java

```diff
@@ -355,6 +355,9 @@ public String format(double number) {
         @Setting(displayName = "Highlight Normal Items", description = "Should normal items be highlighted?", order = 28)
         public boolean normalHighlight = false;
 
+        @Setting(displayName = "Highlight Emeralds in chests", description = "Should emeralds be highlighted in chests?", order = 29)
+        public boolean emeraldHighlightInChest = true;
```

Comment:
False by default, not useful for a lot of players, and might get a bit too overwhelming! The player can easily opt to toggle it on if needed

```suggestion
        public boolean emeraldHighlightInChest = false;
```